check_signed_off.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -x
+
+echo "Checking range: ${TRAVIS_COMMIT_RANGE}"
+
+for i in $( (git rev-list --no-merges "$TRAVIS_COMMIT_RANGE") )
+do
+    COMMIT_MSG="$(git show -s --format=%B "$i")"
+    COMMIT_USER_EMAIL="$(git show -s --format="%an <%ae>" "$i")"
+    echo "$COMMIT_MSG" | grep "^Signed-off-by: ${COMMIT_USER_EMAIL}" >/dev/null
```

Comment:
One last nitpick (I promise it's the last one!): Emails can contain some characters that may have special meaning in regexes, `.` in particular. I would use fgrep instead of grep to grep for the exact string. Then you have to lose the `^` in the beginning, since it will not be a regex anymore, but I think this is acceptable. What do you think?